### ISSUE TITLE
addprocs(SGEManager) fails

### ISSUE THREAD

**Issue by Moelf** [JuliaParallel/ClusterManagers.jl#158]
```
julia> using ClusterManagers, Distributed

julia> addprocs(SGEManager(3,"tier3"))
Error launching workers
KeyError(:qsub_env)
Int64[]

julia> addprocs_sge(3; queue="tier3")
job id is 9500639, waiting for job to start ...
```

**Comment by kescobo** [comment]
Is this recent? Possible regression from #153 ?

**Comment by Moelf** [comment]
this is before the #153 is merged.

**Comment by kescobo** [comment]
Maybe wishful thinking, but is it still happening with #153 ?

**Comment by Moelf** [comment]
yeah, still does, but a different error now haha:
```
julia> manager = SGEManager(3, "tier3", pwd())
SGEManager(3, "tier3", "/home/net3/jiling/qsub")

julia> addprocs(manager)
Unable to read script file because of error: error opening tier3: No such file or directory
Error launching workers
ErrorException("")
Int64[]
```

**Comment by Moelf** [comment]
closed by #160
